src/toCamelCase/index.js

```diff
@@ -0,0 +1,34 @@
+import isPlainObject from 'lodash.isplainobject';
+import camelCase from 'lodash.camelcase';
+import isArray from 'lodash.isarray';
+
+const needsCamelCase = str => {
+	return str.indexOf("-") > -1 || str.indexOf("_") > -1;
+};
+
+const toCamelCase = obj => {
+	Object.keys(obj).forEach(key => {
+		const camelCasedKey = needsCamelCase(key) ? camelCase(key) : key;
+		const value = obj[key];
+		delete obj[key];
+		obj[camelCasedKey] = value;
+		
+		if (isPlainObject(value)) {
+			obj[camelCasedKey] = toCamelCase(value);
+		}
+		
+		if (isArray(value)) {
+			obj[camelCasedKey] = value.map(item => {
+				if (isPlainObject(item)) {
+					return toCamelCase(item);
+				} else {
+					return item;
+				}
+			});
+		}
+	});
+	
+	return obj;
+};
+
+export default toCamelCase;
```

Comment:
Can you please make this a named export with the name matching the filename?  (Related to other comment, let's name is `toCamelLower`)